$OpenBSD: patch-certs_Makefile,v 1.2 2012/10/08 09:11:35 jasper Exp $

- set correct Make program for this GNU Makefile
- make sure the permissions are restricted for the .key files

--- certs/Makefile.orig	Sat Sep 14 14:26:24 2013
+++ certs/Makefile	Sat Sep 14 14:27:56 2013
@@ -2,13 +2,13 @@
 keysize=2048
 
 # How to:
-# First, `make yourhost.cnf` which creates a openssl config file.
+# First, `${MAKE_PROGRAM} yourhost.cnf` which creates a openssl config file.
 # Then edit this file and fill in the details you want it to have,
 # and add or change hosts and components it should cover.
-# Then `make yourhost.key` to create your private key, you can
+# Then `${MAKE_PROGRAM} yourhost.key` to create your private key, you can
 # include keysize=number to change the size of the key.
-# Then you can either `make yourhost.csr` to generate a certificate
-# signing request that you can submit to a CA, or `make yourhost.crt`
+# Then you can either `${MAKE_PROGRAM} yourhost.csr` to generate a certificate
+# signing request that you can submit to a CA, or `${MAKE_PROGRAM} yourhost.crt`
 # to generate a self signed certificate.
 
 .PRECIOUS: %.cnf %.key
@@ -28,3 +28,4 @@ keysize=2048
 %.key:
 	openssl genrsa $(keysize) > $@
 	@chmod 400 $@
+	@chown _prosody $@
